Sidekick.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Version=0.6
```

Comment:
I believe we have to change this based on the version on the gitaction

Reply:
isso é um problema pois ele só aceita versões de dois dígitos (0.0) e não de três (0.0.0)

Por em quanto eu posso deixar na versão 0.1, o que você acha ?